apps/google-play-music-desktop-player/google-play-music-desktop-player.yml

```diff
@@ -28,3 +28,4 @@ locales:
   - ua
 screenshots:
   - imageUrl: 'https://user-images.githubusercontent.com/2289/31847914-51585a3c-b5da-11e7-8082-4a8e62c12b77.gif'
+disabled: true # Dead link(s): https://www.googleplaymusicdesktopplayer.com/
```

Comment:
This one still loads under `http://` 😅

Reply:
I see the issue about the broken ssl yeah! I'll remove the disabled here 👍